src/uwtools/tests/fixtures/hello_workflow_tags.yaml

```diff
@@ -0,0 +1,12 @@
+  workflow:
+    attrs:
+      realtime: false
+      scheduler: slurm
+      cyclethrottle: 5
+    cycledefs:
+      howdy: 
+        - 202209290000 202209300000 06:00:00
```

Comment:
I wonder if its mad about no heading line and the dash?

Reply:
Fixed this with Christina's solution to the `properties` key missing and changing the type from `string` to `array`